package.json

```diff
@@ -105,12 +104,11 @@
     "test:types": "tsc -p tsconfig.spec.json",
     "build": "yarn compile:production && yarn run pack",
     "pack": "electron-builder --publish never",
-    "compile": "yarn run compile:deps && webpack",
+    "compile": "webpack",
     "compile:production": "yarn compile --mode production",
     "compile:web": "webpack --config webpack/renderer.config.js",
     "compile:web:production": "yarn compile:web --mode production",
-    "compile:main": "yarn run compile:deps && webpack --config webpack/main.config.js",
-    "compile:deps": "electron-rebuild",
+    "compile:main": "webpack --config webpack/main.config.js",
```

Comment:
Have you checked the electron binaries? Not sure how the builder handles ensuring the binary is correct for the build? If this is not correct then the flashing just won't work.

Reply:
I was under the impression it was not really needed anymore (read that somewhere), but you're right, I did not really check, so let's keep it.